git/buildable-commits.md

```diff
@@ -0,0 +1,5 @@
+# Buildable commits
+
+We should consider having each commit leaving our code buildable, although we recognize that this might not be possible in every instance.
```

Comment:
To be clear, we're talking about every commit or every PR?
We could make sure every PR is buildable then squash to make master trunk buildable at every commit.

Reply:
we're going to have a meeting to discuss this one